config/settings/development.yml

```diff
@@ -1 +1,4 @@
+# Uncomment to allow sandbox to connect to sul-purl-stage
+# purl_url: "https://sul-purl-stage.stanford.edu"
+
```

Comment:
You need to point to stacks-stage as well for this to work.

Reply:
I believe that's only necessary for Media.

Reply:
Nope, necessary for both.
When you click on the second item in a PDF viewer without the stacks url you get this:
![Screenshot 2025-02-06 at 11 50 59 AM](https://github.com/user-attachments/assets/2295118b-7427-4b6b-bceb-b111ca2a01b7)
If you set the stacks_url you get this:
![Screenshot 2025-02-06 at 11 51 12 AM](https://github.com/user-attachments/assets/47c8632d-57d9-4865-af4b-20623ca2f3bb)